pkg/utils/strings/strings.go

```diff
@@ -234,3 +234,16 @@ func TrimQuotes(s string) string {
 	}
 	return s
 }
+
+func FormalizeWorkingCommand(clusterName string, imageName string, t string, cmd string) string {
```

Comment:
`stringsutil` only contains utils related to strings ops. Do not move `formalizeWorkingCommand` into this package.

Reply:
thanks for the feedback! I unsure about the best location for **formalizeWorkingCommand**. any suggestions?